include/cantera/oneD/IonFlow.h

```diff
@@ -0,0 +1,127 @@
+//! @file IonFlow.h
+
+// This file is part of Cantera. See License.txt in the top-level directory or
+// at http://www.cantera.org/license.txt for license and copyright information.
+
+#ifndef CT_IONFLOW_H
+#define CT_IONFLOW_H
+
+#include "Domain1D.h"
+#include "cantera/base/Array.h"
+#include "cantera/thermo/IdealGasPhase.h"
+#include "cantera/kinetics/Kinetics.h"
+#include "cantera/oneD/StFlow.h"
```

Comment:
minor issue, but since you include `StFlow.h`, you get the rest of these includes automatically.